translate/v3/batch_translate_text.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package v3 contains samples for Google Cloud Translation API v3.
+package v3
+
+// [START translate_v3_batch_translate_text]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	translate "cloud.google.com/go/translate/apiv3"
+	translatepb "google.golang.org/genproto/googleapis/cloud/translate/v3"
+)
+
+// batchTranslateText translates a large volume of text in asynchronous batch mode.
+func batchTranslateText(
+	w io.Writer,
+	projectID string,
+	location string,
+	inputURI string,
+	outputURI string,
+	sourceLang string,
+	targetLang string,
+) error {
+	// projectID := "your-google-cloud-project-id"
```

Comment:
```suggestion
	// projectID := "my-project-id"
```

(Matches the pattern of other samples.)